### DISCUSSION TITLE
Enable TextInput to have array values

### DISCUSSION THREAD

**Discussion by aprilmintacpineda** [mantinedev/mantine#2312] [Feature requests]
Basically like this one:

![image](https://user-images.githubusercontent.com/21032419/187899984-32a8f207-c401-4513-8e3f-9947358f6879.png)

I'm looking at the docs and I haven't seen one so far.

So I'm picturing something similar to `MultiSelect` except it won't have the `data` props.

```tsx
<TextInput
  value={values}
  error="Please enter at least 3 emails"
  onChange={(values) => {
    // values here is string[]
    // e.g., ['me@email.com', 'you@email.com']
    setValues(values);
  }}
  label="Emails"
/>;
```

**Comment by rtivital** [comment]
TagInput component is in backlog, it will be added in one of the next releases
